modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro

```diff
@@ -141,9 +141,11 @@ definition {
 	}
 
 	macro assertEmptyStateTextOnViewBuilder {
+		SelectFrame(locator1 = "ObjectCustomViews#CUSTOM_VIEW_MODAL");
+
 		AssertElementPresent(
 			key_text = "${text}",
-			locator1 = "ObjectCustomViews#VIEW_BUILDER_EMPTY_STATE_TEXT");
+			locator1 = "ObjectCustomViews#CUSTOM_VIEW_MODAL");
```

Comment:
I'm not sure if this fix is correct. The macro is checking for the empty state text on the view builder but this fix just asserts that the custom view modal is present. Can you explain why we're asserting this instead?